sycl/test/esimd/lsc.cpp

```diff
@@ -50,10 +50,10 @@ SYCL_ESIMD_FUNCTION SYCL_EXTERNAL void foo(AccType &acc) {
   simd<uint32_t, VL> offsets = simd<uint32_t, VL>(1) * sizeof(int);
 
   // CHECK: call void @llvm.genx.lsc.store.stateless.v4i1.v4i64.v4i32(<4 x i1> {{[^)]+}}, i8 4, i8 0, i8 0, i16 1, i32 0, i8 3, i8 1, i8 1, i8 0, <4 x i64> {{[^)]+}}, <4 x i32> {{[^)]+}}, i32 0)
-  lsc_scatter<int>(ptr, offsets, data1);
+  lsc_scatter<int, 1>(ptr, offsets, data1);
```

Comment:
In this case invocation became more verbose. Requirement to always add 1 seems very inconvenient. Can this be left as it was?

Reply:
is it possible to modify so that we just write `lsc_scatter(ptr, offsets, data1);` ?